cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -37,8 +37,8 @@
 #
 #  Frontend specific targets:
 #
-#   `openvino::frontend::manager`
-#   OpenVINO frontend manager
+#   `openvino::frontend::common`
+#   OpenVINO frontend common
```

Comment:
I suggest to create alias for manager:
```cmake
add_library(openvino::frontend::manager ALIAD GLOBAL openvino::frontend::common)
```
In this case if user wants to load ANY models, he can link with `openvino::frontend::manager`

@ilyachur how do you think?